revscoring/datasources/revision_oriented.py

```diff
@@ -48,6 +48,7 @@ def __init__(self, name,
                  include_user_last_revision=False,
                  include_page=True,
                  include_page_creation=False,
+                 include_page_suggested=False,
```

Comment:
Style thing, the argument order is different here and below.  Harmless because using keyword params.